src/main/java/brickbreaker/neuralnetworks/Simulation.java

```diff
@@ -0,0 +1,29 @@
+package brickbreaker.neuralnetworks;
+
+import basicneuralnetwork.NeuralNetwork;
+import brickbreaker.model.Game;
+
+import javax.swing.Timer;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.*;
+
+public class Simulation {
+    private Game game;
```

Comment:
Merge Game and Simulation

Reply:
We should combine these two classes into one?

Reply:
I think so.

Reply:
Ok, thank you.